### ISSUE TITLE
Removing Date and friends

### ISSUE THREAD

**Issue by marcverhagen** [lapps/vocabulary-pages#78]
Is the upgrade to 1.2.0 the right time to remove Date etcetera from the hierarchy?

**Comment by ksuderman** [comment]
I would say no.  Since removing things from the vocabulary is a major breaking change they should be removed in the 2.0.0 release.

**Comment by marcverhagen** [comment]
There are two kinds of removing by the way. One is to not print Date et al in the hierarchy but keep their URLs and the other is to remove the from the hierarchy  and remove the URL. I was thinking of the former.

**Comment by keighrim** [comment]
The first way of removing them (hiding from the hierarchy while keeping generated HTML pages) might confuse developers. By having them straight-lined on the visual web site, we can show them explicitly as "deprecated". And to prevent any developer from using them in services.

**Comment by marcverhagen** [comment]
Yes, I can agree with that. I moved this issue to the 2.0.0 milestone.
